Tools/MonoGame.Content.Builder.Editor/Common/PipelineController.cs

```diff
@@ -11,6 +11,7 @@
 using System.Reflection;
 using System.Text;
 using System.Threading.Tasks;
+using Eto.Forms;
```

Comment:
Don't reference Eto.Forms from PipelineController, instead of `Application.Instance.Invoke` call `View.Invoke`.

Reply:
Fixed